sdk/go/common/workspace/plugins.go

```diff
@@ -824,7 +830,8 @@ func (info PluginInfo) Install(tgz io.ReadCloser, reinstall bool) error {
 				return errors.Wrap(err, "installing plugin dependencies")
 			}
 		case "python":
-			if err := python.InstallDependencies(finalDir, "venv", false /*showOutput*/); err != nil {
+			// TODO: pass a context here
```

Comment:
You are passing a context here? Why the TODO? Was this meant for the nodejs case above?

Reply:
I'm not 100% sure, but I should do that. Sorry about that.